torchgeo/transforms/transforms.py

```diff
@@ -62,10 +63,20 @@ def forward(self, batch: Dict[str, Tensor]) -> Dict[str, Tensor]:
             dtype[key] = batch[key].dtype
             batch[key] = batch[key].float()
 
+        # Convert shape of boxes from [N, 4] to [N, 4, 2]
```

Comment:
Is it possible to do this in the individual datasets instead of `AugmentationSequential`? We're trying to get rid of this entirely and use the Kornia implementation in the future. Kornia has keys for bbox, bbox_xyxy, and bbox_xywh to choose from. Also see #985 where I'm trying to develop a standard that matches Kornia. Since there are so few object detection datasets, we should try to remove anything specific to boxes from this function to help slowly phase it out.

Reply:
Ping on this question.

Reply:
Perhaps but I'm not sure why that's needed at this stage. Essentially this line is to reshape the boxes before running the augmentations. This PR adds the same for instance masks as well. This is in addition to the line that was already there for default (semantic segmentation) masks. So if were to phase out this function all three would have to be removed. The standardization of sample keys is a good idea but I think it might be better suited in a separate PR.

The `[N,4,2]` format for boxes is only needed while doing augmentations with kornia. Everywhere else (like plotting or calculating MAP via torchmetrics) we use `[N, 4]`. Hence why this was done here.

Reply:
Alright, if Kornia and torchmetrics also disagree on this, then I'm fine with this hack. As long as we're choosing one of the two standards.